app/Http/Controllers/LockerClaimController.php

```diff
@@ -130,17 +127,99 @@ public function destroy($id)
     public function setup(string $lockerGuid, int $claimId, LockerClaimUpdateRequest $request)
     {
         $lockerClaim = LockerClaim::findOrFail($claimId);
+        $activeClaim = $lockerClaim->locker->activeClaim();
 
-        if (!$lockerClaim->locker->isCurrentlyClaimable()) {
+        if (
+            $this->isOtherClaim($lockerClaim->id, $activeClaim->id) &&
```

Comment:
If `$activeClaim` is `null`, it will return an error because you cannot call `id` from `null`.

Reply:
Switching the logic should work: first check if locker is active, then if it matches the claim.

(My bad.)